app/src/main/kotlin/com/simplemobiletools/musicplayer/activities/SettingsActivity.kt

```diff
@@ -135,6 +138,15 @@ class SettingsActivity : SimpleActivity() {
         }
     }
 
+    private fun setupManualScanning() {
```

Comment:
I dont understand why is this a checkbox in the app settings? It should be a top menu item in the main activity

Reply:
Because it is a switch, not a one-time action. 
Basically, when `Scan files manually` is checked, we'll always show contents from MediaStore and storage files.
When it is disabled, we'll only show content from MediaStore. The preference can be renamed to `Scan all folders` too if that fits better.

Reply:
that is really confusing :/ The user shouldnt have to care how we fetch the files, lets just show whatever we find

Reply:
our users are casual people, not developers. They dont know what is a mediastore or how the app works under the hood, so keep it user friendly.

Reply:
right, I just took the idea of this preference from some other apps where users can enable it if they are having issues with files not showing up. We can just get rid of the preference and always scan everything. The only drawback is that it'll take a little more time to finish the scan. 

> added a new string rescan_media

I made it so that content is always refreshed `onResume()`, do you still think we should add that `Rescan media` menu? or should we disable rescanning in `onResume()` and let the user rescan?

Reply:
lets rescan it when the user presses the top menu for now

Reply:
okay, removing the `Scan files manually` preference and using `rescan_media` string for the menu.